Wikipedia/iOS Native Localizations/ku-latn.lproj/Localizable.strings

```diff
@@ -1,100 +0,0 @@
-"about-libraries-license" = "Destûr";
```

Comment:
Only removed from iOS Native Localizations. It remains in the twn localization folder so the values won't be deleted.